src/main/resources/com/wangyin/parameter/WHideParameterValue/rebuild.jelly

```diff
@@ -0,0 +1,12 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
+         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
+  <j:set var="escapeEntryTitleAndDescription" value="false"/>
+  <f:entry title="${h.escape(it.name)}" description="${it.formattedDescription}">
+    <div name="parameter" description="${it.description}">
+      <input type="hidden" name="name" value="${it.name}"/>
```

Comment:
This is incorrect I think

https://github.com/jenkinsci/hidden-parameter-plugin/blob/main/src/main/resources/com/wangyin/parameter/WHideParameterValue/value.jelly#L7

```xml
	<f:entry title="${h.escape(it.name)}" description="${it.formattedDescription}">
		<f:textbox name="value" value="${it.value}" readonly="true" />
    </f:entry>
```

When testing I was able to edit the hidden parameter. (Which is unwanted it you don't have configure permission)

<img width="978" height="407" alt="edit" src="https://github.com/user-attachments/assets/c6ec9018-fc0e-4e5b-a1ff-2e6c3b6d7f3f" />

The rest looks fine, but I think this edition view is unwanted.

Reply:
I think the textbox with readonly should work

Reply:
Thanks I confirm that the text field is read only now

I will merge and release it tomorrow

Thanks for your PR